atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -560,6 +561,13 @@ def get_build(self, metadata):
                 else:
                     extra['filesystem_koji_task_id'] = task_id
 
+        help_result = self.workflow.prebuild_results.get(AddHelpPlugin.key)
+        if help_result is not None:
+            try:
+                extra['help'] = self.workflow.prebuild_plugins_conf[AddHelpPlugin.key]['help_file']
+            except:
+                extra['help'] = DEFAULT_HELP_FILENAME
```

Comment:
Although it appears to be the correct place because we have existing keys there, I think technically it ought to live in extra['image']['help'].